Access-VBA/articles/881004c6-2dd7-55f1-2a16-2d28034125a8.md

```diff
@@ -19,7 +19,7 @@ The  **SendObject** method carries out the **SendObject** action in Visual Basic
 |:-----|:-----|:-----|:-----|
 |ObjectType|Optional| **AcSendObjectType**|An [AcSendObjectType](36c260a4-69a5-becd-f9f9-7b0346c983df.md) constant that specifies the type of object to send.|
 |ObjectName|Optional| **Variant**|A string expression that's the valid name of an object of the type selected by the ObjectType argument. If you want to include the active object in the mail message, specify the object's type with theObjectType argument and leave this argument blank. If you leave both theObjectType andObjectName arguments blank (the default constant, **acSendNoObject**, is assumed for the ObjectType argument), Microsoft Access sends a message to the electronic mail application without an included database object. If you run Visual Basic code containing the **SendObject** method in a library database, Microsoft Access looks for the object with this name first in the library database, then in the current database.|
-|OutputFormat|Optional| **Variant**||
```

Comment:
A constant that specifies the format in which to send the object. Possible values are **acFormatHTML**, **acFormatRTF**, **acFormatSNP**, **acFormatTXT**, **acFormatXLS**, **acFormatXLSB**, **acFormatXLSX**, **acFormatXPS**, and **acFormatPDF** |